docopt.py

```diff
@@ -462,8 +462,15 @@ def parse_defaults(doc):
 
 
 def parse_section(name, source):
-    pattern = re.compile('^([^\n]*' + name + '[^\n]*\n?(?:[ \t].*?(?:\n|$))*)',
```

Comment:
`[^\n]` were replaced with just `.` because according to the [Regular Expression Syntax](https://docs.python.org/2/library/re.html#regular-expression-syntax):
> (Dot.) In the default mode, this matches any character except a newline.